lobbyboy/utils.py

```diff
@@ -83,10 +83,23 @@ def humanize_seconds(seconds: int):
     return str(timedelta(seconds=seconds))
 
 
-def send_to_channel(channel: Channel, msg: str = "", prefix: str = "", suffix: str = "\r\n"):
-    if isinstance(msg, bytes):
-        msg = msg.decode()
-    channel.send(f"{prefix or ''}{msg}{suffix or ''}".encode())
+def ensure_bytes(s: Union[str, bytes]) -> bytes:
+    if isinstance(s, str):
+        return s.encode()
+    return s
+
+
+def send_to_channel(
+    channel: Channel, msg: Union[str, bytes] = b"", prefix: Union[str, bytes] = b"", suffix: Union[str, bytes] = b"\r\n"
+):
+    msg = ensure_bytes(msg)
+    prefix = ensure_bytes(prefix)
+    suffix = ensure_bytes(suffix)
+
+    buf = bytearray(prefix)
```

Comment:
[Where are python bytearrays used?](https://stackoverflow.com/a/9099768)

Reply:
TIL, but to be that doesn't make a very big difference right? only 3 operations here.

Reply:
Yes, it can make a difference if we reuse the buffer.